workflows/wf3/3-save-options.js

```diff
@@ -20,6 +28,10 @@ fn(state => {
       JSON.stringify({
         optionSets,
         f01MhpssBaseline,
+        f02MhpssFollowUp,
```

Comment:
hmm, I would argue that util.encode should take an object and stringify it for you - since presumably that's 90% of the use-cases.

Can we raise an issue for this too?

Reply:
I don't think it does, i have opened an issue here https://github.com/OpenFn/adaptors/issues/794